skimage/transform/steerable.py

```diff
@@ -97,8 +104,12 @@ def build_scf_pyramid(self, im):
         assert_nD(im, 2)
 
         im = img_as_float(im)
-
         M, N = im.shape
+
+        if min(M, N) < 2**self.height:
+            raise ValueError("Image is not big enough to "
+                             "apply Steerable decomposition")
```

Comment:
I would expand this message to include some helpful specifics such as:

`"For a  {} level decomposition, all image dimensions must be at least size {}.".format(height, 2**height)"`